includes/js/komoju-checkout-blocks.js

```diff
@@ -2,43 +2,41 @@ const KomojuPaymentModule = (() => {
     const { useEffect, useCallback, useRef, createElement } = window.wp.element;
 
     function registerPaymentMethod(paymentMethod) {
-        let name = `${paymentMethod.id}`
+        const name = `${paymentMethod.id}`
         const settings = window.wc.wcSettings.getSetting(`${name}_data`, {});
-
-        const komojuFields =
-            settings.inlineFields
-                ? createElement('komoju-fields', {
-                    'token': '',
-                    'name': 'komoju_payment_token',
-                    'komoju-api': settings.komojuApi,
-                    'publishable-key': settings.publishableKey,
-                    'session': settings.session,
-                    'payment-type': settings.paymentType,
-                    'locale': settings.locale,
-                    style: { display: 'none' },
-                })
-                : null;
+        const description = window.wp.htmlEntities.decodeEntities(settings.description || window.wp.i18n.__('title', 'komoju_woocommerce'));
+        const descriptionDiv = createElement('div',
```

Comment:
Could we set `descriptionDiv` to `null` if `description` is falsy (i.e., zero-length string), maybe?

AIUI `description` can be empty, notably due to the default configurations for plugin versions <= 3.0.9. In such cases `descriptoinDiv` will be an empty `<div>`, which just adds a void blank line under the payment name - I fear that doesn't look good.

Reply:
I set `descriptionDiv` to null when it is empty.
I  also changed the server-side behaviour how it sets the description. Previously, it was setting default description when the description is empty. Now user are able to empty description.

Since description div & komojuField can be null, null checks have been added in the code too.